src/strategies/erc721-pair-weights/README.md

```diff
@@ -0,0 +1,38 @@
+# ERC721 Weighted Pairs
+
+This strategy determines the voting powers for a pair of ERC721 NFT registries. This strategy supports 2 different NFT collections and it provides different voting powers to the holders of these collections, and also supports bonus powers for holders which have items from both the collections.
+
+```json
+{
+    "symbol": "OCMONK",
+    "registries": [
+      "0x960b7a6bcd451c9968473f7bbfd9be826efd549a",
+      "0x86cc280d0bac0bd4ea38ba7d31e895aa20cceb4b"
+    ],
+    "weights": [
+      2,
+      3
+    ],
+    "pairWeight": 4 
+ }
+```
+
+In `example.json`, address `0xE052113bd7D7700d623414a0a4585BCaE754E9d5` has 31 items of `collection1` NFT and 35 items of `collection2` NFT. 
+
+As we can see, there are 31 pairs, 0 items exclusively for collection1 and 4 items exclusively for collection2. So, the voting power of the address will be 
+
+```
+(31 * pairWeight) + (collection1 * weights[0]) + (collection2 * weights[1])`
```

Comment:
There is a bit confusion with strategy name and calculation here 😄 main aim is to add weight for two different collections, but here main priority is given to pair weight than collect weight

Reply:
Yes, the idea was that someone holding from both the collections should have more priority than the addresses that hold items from only one of the two collections. If someone wants no additional priority for the addresses holding pairs, we can assign the values as `pairWeight*2 = weights[0] = weights[1]`

Reply:
it's supposed to give a "bonus" vote(s) if you hold BOTH a genesis and a karma NFT. and more pairs is more bonus. depends on parameters. we can even set weights to penalize pair holders. but we don't expect to use that in our case.